Self hosted docs logan (#1625)
tinacms/tina.io: content/docs/self-hosted/database-adapter/vercel-kv.md

---
title: Vercel KV Database Adapter (Upstash Redis)
id: '/docs/self-hosted/database-adapter/vercel-kv'
prev: '/docs/self-hosted/database-adapter/overview'
next: '/docs/self-hosted/database-adapter/mongodb'
---

The vercel KV database adapter allows you to store your data in the [Vercel KV](https://vercel.com/docs/concepts/projects#environment-variables). This adapter uses the [upstash redis client](https://www.npmjs.com/package/@upstash/redis) so will work on [upstash redis](https://docs.upstash.com/redis) as well.

To get started you will need to add the following environment variables to your project:

> 

Reviewer: Might be nice to have some screenshots to show where to find these values

---
title: Vercel KV Database Adapter (Upstash Redis)
id: '/docs/self-hosted/database-adapter/vercel-kv'
prev: '/docs/self-hosted/database-adapter/overview'
next: '/docs/self-hosted/database-adapter/mongodb'
---

> The vercel KV database adapter allows you to store your data in the [Vercel KV](https://vercel.com/docs/concepts/projects#environment-variables). This adapter uses the [upstash redis client](https://www.npmjs.com/package/@upstash/redis) so will work on [upstash redis](https://docs.upstash.com/redis) as well.

Reviewer 2: ```suggestion
The Vercel KV database adapter allows you to use the [Vercel KV](https://vercel.com/docs/storage/vercel-kv) database with Tina. This adapter uses the [upstash redis client](https://www.npmjs.com/package/@upstash/redis), allowing it to also work with [upstash redis](https://docs.upstash.com/redis) as well.
```

wh

> To get started you will need to setup some en  environment variables.

Reviewer 2: ```suggestion
To get started, you will need to setup some environment variables.
```
